testsuite/scripts/gha_test_run_proto.json

```diff
@@ -8,7 +8,13 @@
             "suite":"TestBus",
             "test":"test_get",
             "skip":true,
-            "skipReason":"UnitTest sometimes fails when run in qpm (FIXME)"
+            "skipReason":"UnitTest fails when run in qpm (FIXME)"
+        },
+        {
+            "suite":"TestBus",
```

Comment:
`TestBus` are weird. They show as if they fail, but they actually don't register as a failure in the end. So I don't think they need to be skipped (as they weren't before), this is something to address in the testing framework itself...

Reply:
I was seeing a failure on macOS (or linux?) earlier. I think the `test_get` and `test_getn` are ones we should skip just to be safe.

Reply:
My personal preference would be not to touch tests unrelated to the CondVar change. `TestBus` passes at least sometime:
https://github.com/supercollider/supercollider/runs/2651269477?check_suite_focus=true#step:8:14863
https://github.com/supercollider/supercollider/runs/2651269468?check_suite_focus=true#step:9:15393

But if you want really want to keep this change, this fine too.

Reply:
For this PR, I'd like to get to a point where are running tests pass everytime(-ish). There's more work needed to be done in follow up PRs.